### ISSUE TITLE
Can't load in 1.5

### ISSUE THREAD

**Issue by guihigashi** [chkwon/PATHSolver.jl#42]
Thanks for the package. It seems `jl_function_ptr` was removed. Here's a link with a issue that seems similar and the error message.

https://github.com/JuliaGraphics/Gtk.jl/pull/512

```
ERROR: LoadError: LoadError: could not load symbol "jl_function_ptr":
The specified procedure could not be found.
Stacktrace:
 [1] PATHSolver.FunctionWrappersQuickFix.FunctionWrapper{Any,Tuple{Any}}(::typeof(identity)) at C:\Users\XXXXXXX\.julia\packages\PATHSolver\tJVDr\src\FunctionWrappersQuickFix.jl:89
 [2] top-level scope at C:\Users\XXXXXXX\.julia\packages\PATHSolver\tJVDr\src\FunctionWrappersQuickFix.jl:132
 [3] include(::Function, ::Module, ::String) at .\Base.jl:380
 [4] include at .\Base.jl:368 [inlined]
 [5] include(::String) at C:\Users\XXXXXXX\.julia\packages\PATHSolver\tJVDr\src\PATHSolver.jl:1
 [6] top-level scope at C:\Users\XXXXXXX\.julia\packages\PATHSolver\tJVDr\src\PATHSolver.jl:3
 [7] include(::Function, ::Module, ::String) at .\Base.jl:380
 [8] include(::Module, ::String) at .\Base.jl:368
 [9] top-level scope at none:2
 [10] eval at .\boot.jl:331 [inlined]
 [11] eval(::Expr) at .\client.jl:467
 [12] top-level scope at .\none:3
in expression starting at C:\Users\XXXXXXX\.julia\packages\PATHSolver\tJVDr\src\FunctionWrappersQuickFix.jl:132
in expression starting at C:\Users\XXXXXXX\.julia\packages\PATHSolver\tJVDr\src\PATHSolver.jl:3
ERROR: LoadError: Failed to precompile PATHSolver [f5f7c340-0bb3-5c69-969a-41884d311d1b] to C:\Users\XXXXXXX\.julia\compiled\v1.5\PATHSolver\xyGrF_xqMiq.ji.
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at .\loading.jl:1290
 [3] _require(::Base.PkgId) at .\loading.jl:1030
 [4] require(::Base.PkgId) at .\loading.jl:928
 [5] require(::Module, ::Symbol) at .\loading.jl:923
 [6] include(::Function, ::Module, ::String) at .\Base.jl:380
 [7] include(::Module, ::String) at .\Base.jl:368
 [8] top-level scope at none:2
 [9] eval at .\boot.jl:331 [inlined]
 [10] eval(::Expr) at .\client.jl:467
 [11] top-level scope at .\none:3
in expression starting at C:\Users\XXXXXXX\.julia\packages\Complementarity\jpfgn\src\Complementarity.jl:1
```

**Comment by chkwon** [comment]
Thanks for reporting. Please consider this package: https://github.com/odow/PATH.jl
Eventually, Complementarity.jl will phase out and PATH.jl (not yet officially registered package) will kick in.

**Comment by chkwon** [comment]
@guihigashi  This issue should have been fixed now with the new version v0.6.2. Please test it and let me know any issues.
